src/components/DocumentList.js

```diff
@@ -0,0 +1,54 @@
+import { displayDocumentList } from "../utils/displayDocumentList.js";
+import { push } from "../utils/router.js";
+import { setItem, getItem, removeItem } from "../utils/storage.js";
+
+export default function PostList({ $target, initialState }) {
+  const $documentList = document.createElement("div");
+  $documentList.className = "document-list";
+  $target.appendChild($documentList);
+
+  this.state = initialState;
+
+  this.setState = (nextState) => {
+    this.state = nextState;
+    this.render();
+  };
+
+  this.render = () => {
+    $documentList.innerHTML = `
+            <button name ="add-btn" data-id = "null" class="root-page-add" >페이지 추가</button>
+            <div class ="list-box">
+            ${displayDocumentList(this.state)}
+            </div>
+        `;
+  };
+
+  this.render();
+
+  $documentList.addEventListener("click", (e) => {
+    const { target } = e;
+    const element = target.closest("[name]");
```

Comment:
p4: 이런식으로도 바꿀수 있을것 같습니다! 그리고 element라는 변수명은 약간 추상적일수 있다고 생각하는데 조금더 구체적으로 네이밍하면 어떨까요?
```suggestion
  $documentList.addEventListener("click", ({target}) => {
    const element = target.closest("[name]");
```